### DISCUSSION TITLE
How do i decrease number width ? [new to neovim]

### DISCUSSION THREAD

**Discussion by aarsh21** [LazyVim/LazyVim#4238] [Q&A]
![image](https://github.com/user-attachments/assets/eb3a5d14-b14e-439c-b082-cda3442e6f61)

**Comment by smarquez1** [comment]
You need to remove or customize the [statuscolumn](https://neovim.io/doc/user/options.html#'statuscolumn'), or just remove its contents. Try these options:

```vim
set statuscolumn = ""
set numberwidth = 0
set norelativenumber
set nonumber
```

BTW: Here is is Lazyvims statuscolumn config: https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/util/ui.lua#L94

**Comment by windowsrefund** [comment]
FYI, `numberwidth` must be a positive integer. Setting it to a value of `1` seems like the move.
